CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+## Contribution Guidelines
+
+Have you found a bug? Do you have a new idea for a feature? Want to submit an issue to help us improve the site? Thanks so much! Please use the below format to pass on all the details and increase the chance your issue will be picked up.
+
+### Quicklink: [Submit An Issue](https://github.com/CodeMontageHQ/codemontage/issues/new?body=Description%0A-------%0A%20%0A%0AAsk%0A--------%0A%0ASteps%0A--------%0A%0ATo%20Do%20(Optional)%0A--------%0A%0AAdditional%20Info%20(Optional)%0A--------%0A)
+
+### Description
+Include a high level description of the feature or error here including steps of how to recreate if applicable. Include any benefits, challenges or considerations. Can be short and sweet. Use GitHub-flavored markdown to link any related issues.
+
+### Ask
+Describe the desired behavior and what would deem this issue/bug/feature complete. If there are several options, list them or create separate issues for them.
+
+### To Do (optional)
+
+ Steps
+ To
+ Complete
+ Fix
+ Go in a task list
+
+### Additional Info (optional)
+Include any images, steps to recreate, notes, emojis, or whatever. This may helpfully include code snippets, screenshots, and/or videos of a bug in its natural habitat.
+
+#[Create Issue!](https://github.com/CodeMontageHQ/codemontage/issues/new?body=Description%0A-------%0A%20%0A%0AAsk%0A--------%0A%0ASteps%0A--------%0A%0ATo%20Do%20(Optional)%0A--------%0A%0AAdditional%20Info%20(Optional)%0A--------%0A)
```

Comment:
I think this is meant to be a header? If so, it needs a space between the octothorpe and the link's opening bracket.